EIPS/eip-2938.md

```diff
@@ -0,0 +1,262 @@
+---
+eip: 2938
+title: Account Abstraction
+author: Vitalik Buterin (@vbuterin), Ansgar Dietrichs (@adietrichs), Matt Garnett (@lightclient), Will Villanueva (@villanuevawill), Sam Wilson (@SamWilsn)
+discussions-to: https://ethereum-magicians.org/t/implementing-account-abstraction-as-part-of-eth1-x/4020
+status: Draft
+type: Standards Track
+category: Core
+created: 2020-09-04
+requires: 1344, 2718
+replaces: 86
```

Comment:
```suggestion
```
You cannot replace/supersede a draft transaction, only a final one.

Note: just saw this at a quick glance, will do a full review later.